code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -31,7 +31,12 @@ namespace <#= qualifiedNamespace #>
             private static readonly ComponentType targetComponentType = typeof(<#= componentDetails.TypeName #>);
 
             public override ComponentType[] ReplicationComponentTypes => replicationComponentTypes;
-            private static readonly ComponentType[] replicationComponentTypes = { typeof(<#= componentDetails.TypeName #>), typeof(Authoritative<<#= componentDetails.TypeName #>>), typeof(SpatialEntityId)};
+            private static readonly ComponentType[] replicationComponentTypes = { typeof(<#= componentDetails.TypeName #>), typeof(Authoritative<<#= componentDetails.TypeName #>>), 
```

Comment:
Make sure you run codegen. The test schema should be in the PR.

Reply:
Ahh yes, added.